setup.cfg

```diff
@@ -120,6 +120,15 @@ ignore_missing_imports = True
 [mypy-urllib3.*]
 ignore_missing_imports = True
 
+[mypy-fetchai_crypto.*]
```

Comment:
why `-` and not `_`?

Reply:
see above? `-` is used to separate `mypy` and the package name, `_` is part of the package name.